components/moneybird/actions/create-contact/create-contact.mjs

```diff
@@ -0,0 +1,68 @@
+import moneybird from "../../moneybird.app.mjs";
+
+export default {
+  key: "moneybird-create-contact",
+  name: "Create Contact",
+  description: "Create a new contact. [See docs here](https://developer.moneybird.com/api/contacts/#post_contacts)",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    moneybird,
+    firstName: {
+      label: "First Name",
+      description: "First name of contact",
+      type: "string",
+    },
+    lastName: {
+      label: "Last Name",
+      description: "Last name of contact",
+      type: "string",
+      optional: true,
+    },
+    companyName: {
+      label: "Company Name",
+      description: "Company name of contact",
+      type: "string",
+      optional: true,
+    },
+    country: {
+      label: "Country",
+      description: "Country of contact",
+      type: "string",
+      optional: true,
+    },
+    city: {
+      label: "City",
+      description: "City of contact",
+      type: "string",
+      optional: true,
+    },
+    sendInvoicesToEmail: {
+      label: "Send Invoices To Email",
+      description: "The email to send the invoices",
+      type: "string",
+      optional: true,
+    },
```

Comment:
Optionally we can add another `object` type prop like `additionalFields` so that users would be enabled to set other fields which are not here, e.g. `tax_number `.